internal/activation/activation.go

```
@@ -688,6 +852,20 @@ func Status(ctx context.Context, root string, runner provider.Runner, repository
 	state.Problems = append(state.Problems, verifyReceipt(ctx, receipt, source, runner)...)
 	if len(state.Problems) > 0 {
 		state.Status = StatusDrifted
+		return state, nil
+	}
+	if receipt.Project != nil {
+		contract, contractErr := FirstUseContract(receipt)
+		if contractErr != nil {
+			state.Smoke = smoke.Evidence{Status: smoke.StatusAwaiting, Problems: []string{contractErr.Error()}}
+			return state, nil
```

golangci-lint (2.12.2)

[error] error is not nil (line 858) but it returns nil

(nilerr)